docs/resources/Channel.md

```diff
@@ -337,6 +337,11 @@ Represents a message sent in a channel within Discord.
 | AUTO_MODERATION_ACTION                       | 24    | true*     |
 | ROLE_SUBSCRIPTION_PURCHASE                   | 25    | true      |
 | INTERACTION_PREMIUM_UPSELL                   | 26    | true      |
+| STAGE_START                                  | 27    | true      |
+| STAGE_END                                    | 28    | true      |
+| STAGE_SPEAKER                                | 29    | true      |
+| STAGE_RAISE_HAND                             | 30    | true      |
```

Comment:
This is client side only and shouldn't be documented. The rest look good though.